client/core/core.go

```diff
@@ -5324,6 +5324,10 @@ func (c *Core) Send(pw []byte, assetID uint32, value uint64, address string, sub
 		return nil, err
 	}
 
+	if !wallet.synchronized() {
+		return nil, fmt.Errorf("%s is still syncing", unbip(assetID))
+	}
```

Comment:
Only other thought is that we could add peers>0 check to this as well.  See other spots where we directly access the xcWallet fields under mutex lock, or use the state method, which is slightly more work.

Reply:
Updated. `postRequiredBonds` does not check for peer count atm but checks for `wallet.synchronized()`, do you want us to also check for peer count there?

Reply:
Let's do it.
I don't like transactions going nowhere

Reply:
Updated my last commit.